src/modules/signer/signer.service.ts

```diff
@@ -502,4 +505,43 @@ export class SignerService {
       sig
     )}`;
   }
+
+  /**
+   * Set `_isEthSigner` value based on a signed message.
+   * Generates a test message and signs it.
+   */
+  private async _setIsEthrSigner() {
+    const header = {
+      alg: 'ES256',
+      typ: 'JWT',
+    };
+    const encodedHeader = base64url(JSON.stringify(header));
+    const address = this._address;
+    const payload = {
+      address,
+    };
+
+    const encodedPayload = base64url(JSON.stringify(payload));
+    const token = `0x${Buffer.from(
+      `${encodedHeader}.${encodedPayload}`
+    ).toString('hex')}`;
```

Comment:
I don't think we this JWT stuff. I think we just need to sign a message and see if we can recover the address. Is there an advantage to the JWT bit?

Reply:
Yeah, you are right. simple message is enough